changelogs/fragments/66792-vultr-improve-plan.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
Shouldn't this be `bugfixes:`?

Reply:
I decided to call it minor change because it fixes an edge case of the api and I would not create a backport it because it is hard to test and probably I missed something.